content/id/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Penyetelan Performa Penjadwal
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.14" state="beta" >}}
+
+[kube-scheduler](/docs/concepts/scheduling/kube-scheduler/#kube-scheduler)
+merupakan penjadwal (_scheduler_) Kubernetes yang _default_, bertanggung jawab
+terhadap penempatan Pod-Pod pada seluruh Node di dalam sebuah klaster.
+
+Node-Node di dalam klaster yang sesuai dengan syarat-syarat penjadwalan dari
+sebuah Pod disebut sebagai Node-Node layak (_feasible_). Penjadwal mencari Node-Node
+layak untuk sebuah Pod dan kemudian menjalankan fungsi-fungsi untuk menghitung
+penilaian (_score_) dari Node-Node tersebut, memilih sebuah Node dengan penilaian tertinggi di antara
+Node-Node layak lainnya, dimana Pod akan dijalankan. Penjadwal kemudian memberitahu
+peladen (_server_) API soal keputusan ini melalui sebuah proses yang disebut _Binding_.
+
+Laman ini menjelaskan optimasi penyetelan (_tuning_) performa yang relevan
+untuk klaster Kubernetes berskala besar.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+Pada klaster berskala besar, kamu bisa menyetel perilaku penjadwal
+untuk menyeimbangkan keluaran penjadwalan antara _latency_ (seberapa cepat Pod-Pod baru ditempatkan)
+dan akurasi (seberapa akurat penjadwal membuat keputusan penjadwalan yang tepat).
+
+Kamu bisa mengkonfigurasi setelan ini melalui pengaturan `percentageOfNodesToScore` pada kube-scheduler.
+Pengaturan KubeSchedulerConfiguration ini menentukan sebuah _threshold_ untuk
```

Comment:
```suggestion
Pengaturan KubeSchedulerConfiguration ini menentukan sebuah ambang batas (_threshold_) untuk
```
got it from here
https://kbbi.kemdikbud.go.id/entri/ambang%20batas

Reply:
fixed in https://github.com/kubernetes/website/pull/20260/commits/3cc163e65705127d1fcf16feba108179f9bdacc2